src/main/java/com/example/notification/IMIconnectApi/service/SmsSender.java

```diff
@@ -0,0 +1,9 @@
+package com.example.notification.IMIconnectApi.service;
+
+import com.example.notification.IMIconnectApi.model.ImiSmsRequest;
+import org.springframework.stereotype.Service;
+
+@Service
+public interface SmsSender {
+    public String smsSend(ImiSmsRequest imiSmsRequest);
```

Comment:
you don't need to explicitly mention public here, interface functions are by default public.